.gitignore

```diff
@@ -43,4 +43,6 @@ docker/sample/domain-admin/domain_admin_cert.pem
 docker/sample/domain-admin/domain_admin_key.pem
 docker/zms/conf/zms.properties
 
-manual-testing/
\ No newline at end of file
+manual-testing/
+
+/example
```

Comment:
pls avoid adding unnecessary folders here unless it is really needed. 
and if it is, consider change it to local-testing or something similar